cli/packages/cmd/secrets.go

```diff
@@ -189,10 +189,24 @@ var secretsSetCmd = &cobra.Command{
 		if token != nil && (token.Type == util.SERVICE_TOKEN_IDENTIFIER || token.Type == util.UNIVERSAL_AUTH_TOKEN_IDENTIFIER) {
 			secretOperations, err = util.SetRawSecrets(args, secretType, environmentName, secretsPath, projectId, token)
 		} else {
-			util.RequireLogin()
-			util.RequireLocalWorkspaceFile()
+			workspaceFile, err := util.GetWorkSpaceFromFile()
```

Comment:
Why did we remove?

```
			util.RequireLogin()
			util.RequireLocalWorkspaceFile()
```

Reply:
Instead i am explicitly doing a check for that and getting the logged in credential